app/models/translation.rb

```diff
@@ -20,11 +20,10 @@ class Translation < ApplicationRecord
   scope :by_created, -> { order("created_at ASC") }
 
   FIELD_TYPES = {
-    "String"    => "string",
-    "Boolean"   => "boolean",
-    "Text"      => "text",
-    "Rich Text" => "rich_text",
-    "Images"    => "images",
+    "String"  => "string",
+    "Boolean" => "boolean",
+    "Text"    => "text",
+    "Images"  => "images",
```

Comment:
wondering if we should map Rich Text to text for backward compatibility with existing forms

Reply:
never mind, saw you reworked :rich_text to :text elsewhere